Library/Homebrew/livecheck/strategy/sparkle.rb

```diff
@@ -92,7 +92,7 @@ def self.items_from_content(content)
             pub_date = item.elements["pubDate"]&.text&.strip&.presence&.then do |date_string|
               Time.parse(date_string)
             rescue ArgumentError
-              # Omit unparseable strings (e.g. non-English dates)
+              # Omit unparsable strings (e.g. non-English dates)
```

Comment:
The original is considered an alternative spelling, but given that the -able suffix starts with a vowel, dropping the silent -e is valid here.